### PR TITLE
Full review of repository code by Daria Hrychyna

### PR DESCRIPTION
**Pull request by DariaHrychyna**

### CODE REVIEW

File: static/kanban.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    function addTask() {
+        const taskText = newTaskInput.value.trim();
+        if (taskText !== '') {
+            const newTask = document.createElement('div');
+            newTask.classList.add('task');
+            newTask.draggable = true;
+            newTask.textContent = taskText;
+            newTask.addEventListener('dragstart', dragStart);
```
Comment (DariaHrychyna) [inline static/kanban.js:61]: Attach the drag and click event listeners to the new task

File: static/kanban.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        this.appendChild(task);
+    }
+
+    function addTask() {
+        const taskText = newTaskInput.value.trim();
+        if (taskText !== '') {
+            const newTask = document.createElement('div');
+            newTask.classList.add('task');
+            newTask.draggable = true;
+            newTask.textContent = taskText;
```
Comment (DariaHrychyna) [inline static/kanban.js:60]: Attach the drag and click event listeners to the new task

File: static/kanban.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            newTask.addEventListener('dragstart', dragStart);
+            newTask.addEventListener('dragend', dragEnd);
+            newTask.addEventListener('click', toggleTaskWindow);
+            columns[0].appendChild(newTask);
+            newTaskInput.value = '';
+        }
+    }
+
+    function toggleTaskWindow() {
+        const description = this.querySelector('.description');
```
Comment (DariaHrychyna) [inline static/kanban.js:70]: Toggle description visibility

File: static/kanban.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+                createDescriptionElement(this);
+            } else {
+                description.classList.remove('hidden');
+            }
+        }
+    }
+
+    function createDescriptionElement(task) {
+        const newDescription = document.createElement('div');
+        newDescription.classList.add('description');
```
Comment (DariaHrychyna) [inline static/kanban.js:85]: Please, consider using template literals for readability

File: static/whiteboard.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+let cellSize = 20; // Initial size of each cell in the grid
+let zoomLevel = 1; // Initial zoom level
+const maxZoom = 3; // Maximum allowed zoom level
+const minZoom = 0.5; // Minimum allowed zoom level
+
+// Function to draw the grid with the current cell size and offset
+function drawGrid() {
+    ctx.clearRect(0, 0, canvas.width, canvas.height); // Clear the canvas
+    const scaledCellSize = cellSize * zoomLevel; // Adjusted cell size based on zoom level
+
```
Comment (DariaHrychyna) [inline static/whiteboard.js:13]: Use a single loop to draw both vertical and horizontal lines

Comment (MaraaaBodnar) [reply static/whiteboard.js:13]: For now I don't know how to do it in a single loop but I am grateful for your consideration

File: static/kanban.js (hunk 4 above)
Comment (MaraaaBodnar) [reply static/kanban.js:85]: I will consider it in the nearest future

**Pull request merged**